### ISSUE TITLE
Not working as of patch v1.4.14 (09/01/22)

### ISSUE THREAD

**Issue by richardmcameron** [nebriv/VTOLVR-TacviewLogger#29]
Upon installing (1 click install) v2.6 of the mod, loading the game and then loading the mod, no TacViewDataLogs folder is generated in the VTOLVR game folder nor is there any data generated upon flights ending. This is on a fresh install with only Tacview Data Logger installed.

**Comment by richardmcameron** [comment]
`Mod Loader: Tacview Data Logger doesn't have a dependencies folder. Checked I:\SteamLibrary\steamapps\common\VTOL VR\VTOLVR_Modloader\mods\Tacview_Data_Logger\dependencies DevFolder = False
Mod Loader: Loading from I:\SteamLibrary\steamapps\common\VTOL VR\VTOLVR_Modloader\mods\Tacview_Data_Logger\TacViewDataLogger.dll
Fallback handler could not load library I:/SteamLibrary/steamapps/common/VTOL VR/VTOLVR_Data/Mono/data-0000014B7E025DC0.dll
Tacview Data Logger: Loaded!
Updating Discord...
Mod Loader: End of LoadMod
PlatformNotSupportedException: Operation is not supported on this platform.
  at System.AppDomain.DefineDynamicAssembly (System.Reflection.AssemblyName name, System.Reflection.Emit.AssemblyBuilderAccess access) [0x00000] in <a1e9f114a6e64f4eacb529fc802ec93d>:0 
  at (wrapper remoting-invoke-with-check) System.AppDomain.DefineDynamicAssembly(System.Reflection.AssemblyName,System.Reflection.Emit.AssemblyBuilderAccess)
  at Harmony.HarmonySharedState.GetState () [0x00039] in <ef472ad41f4948fdb52674edfe069979>:0 
  at Harmony.HarmonySharedState.GetPatchInfo (System.Reflection.MethodBase method) [0x00001] in <ef472ad41f4948fdb52674edfe069979>:0 
  at Harmony.PatchProcessor.Patch () [0x0006e] in <ef472ad41f4948fdb52674edfe069979>:0 
  at Harmony.HarmonyInstance.<PatchAll>b__9_0 (System.Type type) [0x0002c] in <ef472ad41f4948fdb52674edfe069979>:0 
  at Harmony.CollectionExtensions.Do[T] (System.Collections.Generic.IEnumerable`1[T] sequence, System.Action`1[T] action) [0x0001b] in <ef472ad41f4948fdb52674edfe069979>:0 
  at Harmony.HarmonyInstance.PatchAll (System.Reflection.Assembly assembly) [0x00007] in <ef472ad41f4948fdb52674edfe069979>:0 
  at TacViewDataLogger.TacViewDataLogger.Start () [0x0001e] in <2e0e8b74725f44a68ba3843d9aa2e3dc>:0 `

**Comment by richardmcameron** [comment]
Can confirm #28 is still an issue. After a full integrity check / uninstall and reinstall. TacViewDataLogs folder was created and some data was recorded. However, very little is actually captured, leading to very sparse and intermittent playback in tacview. 

![aa6d821402c4d56a8fd3c27e1f169d4a](https://user-images.githubusercontent.com/65136585/189507510-594b4007-ec63-4a82-9bbd-81c818fc4f83.gif)

**Comment by bergsm** [comment]
@nebriv FWIW I was able to get the mod working again by importing the source code into VS and recompiling with the latest .dlls from VTOL VR. Might be an easy fix for you if you want to update. Would also fix #28 potentially.

**Comment by nebriv** [comment]
Yeah it realistically is an easy fix. I just haven't had much motivation to re-setup everything. Thanks for testing it out. I'll see if I can get it rebuilt this week.
